cmd/root.go

```diff
@@ -54,6 +54,11 @@ The default adapter can be overridden per device`,
 		`Communication parameters for default adapter, either 8N1 or 8E1.
 Only applicable if the default adapter is an RTU device`,
 	)
+	rootCmd.PersistentFlags().IntP(
+		"timeout", "t",
+		300,
```

Comment:
Use const here. Do `PersistentFlags` support durations?

Reply:
Yes, it looks like PersistenFlags support durations. 
> Duration flags accept any input valid for time.ParseDuration.

[pflag](https://pkg.go.dev/github.com/spf13/pflag#pkg-overview)